test/ecs.go

```diff
@@ -0,0 +1,117 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build integration
+// +build integration
+
+package test
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
```

Comment:
There should be Go SDK V2 equivalents for this. There might be code changes necessary, but I don't think there's a good reason to include both SDKs

Reply:
will try [TODO]